packages/catalog-server/src/test/lib/catalog_test.ts

```diff
@@ -358,4 +358,42 @@ test('Imports a non-existent package', async () => {
   assert.equal(importResult.packageInfo?.status, PackageStatus.NOT_FOUND);
 });
 
+test.only('Imports a large manifest', async () => {
+  const packageName = 'large-manifest';
+  const packagePath = fileURLToPath(
+    new URL('../test-packages/large-manifest/', import.meta.url)
+  );
+
+  const files = new LocalFsPackageFiles({
+    path: packagePath,
+    packageName,
+    publishedVersions: ['1.0.0'],
+    distTags: {
+      latest: '1.0.0',
+    },
+  });
+  const repository = new FirestoreRepository(TEST_SEQUENCE_THREE);
```

Comment:
How would I know what test sequence to use if I were writing a new test?

Reply:
Good question, and "sequence" might be a bad name. The important part is that tests that rely on the same data need the same namespace so that one test can write data that another reads. That's why I named it "sequence" because all those tests need to run in order. But I'm using TEST_SEQUENCE_THREE incorreclty here because these don't belong to a sequence...